### PR TITLE
CE-33336. The `laminas/laminas-soap` composer dependency has been eliminated

### PR DESCRIPTION
**Pull request by anzin**
### Description (*)
This PR (pull request) provides the eliminating `laminas/laminas-soap` composer dependency from the system.

**Need to close** : https://github.com/magento/magento2/pull/33660

### Related Pull Requests
https://github.com/magento/partners-magento2ee/pull/574

### Fixed Issues (if relevant)
1. Fixes magento/magento2#33336

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run all tests

File: app/code/Magento/Webapi/Api/Data/ClientInterface.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @see       https://github.com/laminas/laminas-server for the canonical source repository
```
Comment (ihor-sviziev) [inline app/code/Magento/Webapi/Api/Data/ClientInterface.php:3]: Copying lamias components to Magento isn't a really good idea.
More details here https://github.com/magento/magento2/pull/33373#discussion_r676573460
I still didn't get any answer.

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run Functional Tests B2B

**Comment by anzin** [comment]
@magento run Unit Tests

**Comment by anzin** [comment]
@magento run Functional Tests B2B

**Comment by anzin** [comment]
@magento run Unit Tests, Static Tests

**Comment by anzin** [comment]
@magento run Unit Tests, Static Tests

**Comment by Den4ik** [comment]
@anzin Could you check EE and B2B unit tests

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run  Functional Tests B2B

**Comment by anzin** [comment]
@magento run Functional Tests B2B

**Comment by anzin** [comment]
@magento run Functional Tests B2B

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run all tests

**Comment by Den4ik** [comment]
@anzin I see same failed Unit test that related to changes. Could you lease check it

**Comment by anzin** [comment]
@magento run Functional Tests B2B, Functional Tests CE, Functional Tests EE, Integration Tests

**Comment by anzin** [comment]
@magento run  Semantic Version Checker

**Comment by anzin** [comment]
@magento run Unit Tests, Functional Tests B2B, Functional Tests CE

**Comment by anzin** [comment]
@magento run Functional Tests B2B, Functional Tests CE

**Comment by anzin** [comment]
@magento run Functional Tests B2B, Functional Tests CE

**Comment by anzin** [comment]
@magento run all tests

**Comment by anzin** [comment]
@magento run WebAPI Tests

**Pull request closed without merging**